### PR TITLE
removing graphite1(vcloud) from graphite nodes

### PR DESCRIPTION
**Pull request by vladimirhasko**

### CODE REVIEW

**Review by vladimirvshivkov — APPROVED** [review]

**Review by vladimirvshivkov — APPROVED** [review]

File: zuul.d/system-config-run.yaml [hunk 1]
```diff
@@ -337,7 +337,7 @@
       - playbooks/bootstrap-bridge.yaml
       - playbooks/service-proxy.yaml
       - playbooks/roles/haproxy
-      - inventory/service/group_vars/proxy.yaml
+      # - inventory/service/group_vars/proxy.yaml
```
Comment (gtema) [inline zuul.d/system-config-run.yaml:340]: Why is that necessary?

File: inventory/service/group_vars/carbonapi.yaml [hunk 2]
```diff
@@ -10,7 +10,6 @@ carbonapi_instances:
         lbMethod: "any"
         protocol: "carbonapi_v3_pb"
         servers:
```
Comment (gtema) [inline inventory/service/group_vars/carbonapi.yaml:12]: Pls double check you remove the correct instance, I was sure it should be graphite2
